sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests/Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.csproj

```diff
@@ -31,9 +32,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\applicationinsights\Microsoft.Azure.ApplicationInsights.Query\src\Microsoft.Azure.ApplicationInsights.Query.csproj" />
```

Comment:
A PacakgeReference would be preferable.

Reply:
haha okay. I started with that, but I assumed we would want a reference to the project.

Reply:
You might need to add a centralized version into the testing sections of Packages.Data.props.